bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/tags/SystemTags.java

```diff
@@ -0,0 +1,82 @@
+        Object,
+        Location,
+        Property,
+        Purpose,
+        Capability;
+    }
+
+    public static final String LOC_HOME = Kind.Location + TAG_KIND_DELIMITER + "Home";
+    public static final String LOC_WORK = Kind.Location + TAG_KIND_DELIMITER + "Work";
+
+    public static final String PROP_TEMPERATURE = Kind.Property + TAG_KIND_DELIMITER + "Temperature";
+    public static final String PROP_LIGHT = Kind.Property + TAG_KIND_DELIMITER + "Light";
+    public static final String PROP_HUMIDITY = Kind.Property + TAG_KIND_DELIMITER + "Humidity";
+    public static final String PROP_PRESSURE = Kind.Property + TAG_KIND_DELIMITER + "Pressure";
+    public static final String PROP_SMOKE = Kind.Property + TAG_KIND_DELIMITER + "Smoke";
+    public static final String PROP_NOISE = Kind.Property + TAG_KIND_DELIMITER + "Noise";
+    public static final String PROP_RAIN = Kind.Property + TAG_KIND_DELIMITER + "Rain";
+    public static final String PROP_WIND = Kind.Property + TAG_KIND_DELIMITER + "Wind";
+    public static final String PROP_WATER = Kind.Property + TAG_KIND_DELIMITER + "Water";
+    public static final String PROP_CO2 = Kind.Property + TAG_KIND_DELIMITER + "CO2";
+    public static final String PROP_CO = Kind.Property + TAG_KIND_DELIMITER + "CO";
+    public static final String PROP_ENERGY = Kind.Property + TAG_KIND_DELIMITER + "Energy";
+    public static final String PROP_POWER = Kind.Property + TAG_KIND_DELIMITER + "Power";
+    public static final String PROP_VOLTAGE = Kind.Property + TAG_KIND_DELIMITER + "Voltage";
+    public static final String PROP_CURRENT = Kind.Property + TAG_KIND_DELIMITER + "Current";
+    public static final String PROP_FREQUENCY = Kind.Property + TAG_KIND_DELIMITER + "Frequency";
+    public static final String PROP_GAS = Kind.Property + TAG_KIND_DELIMITER + "Gas";
+    public static final String PROP_OIL = Kind.Property + TAG_KIND_DELIMITER + "Oil";
+
+    public static final String PUR_ALARM = Kind.Purpose + TAG_KIND_DELIMITER + "Alarm";
+    public static final String PUR_ALARM_FIRE = PUR_ALARM + TAG_KIND_DELIMITER + "Fire";
+    public static final String PUR_ALARM_LEAKAGE = PUR_ALARM + TAG_KIND_DELIMITER + "Leakage";
+    public static final String PUR_ALARM_INTRUSION = PUR_ALARM + TAG_KIND_DELIMITER + "Intrusion";
+    public static final String PUR_ALARM_CO = PUR_ALARM + TAG_KIND_DELIMITER + "CO";
+    public static final String PUR_LIGHTING = Kind.Purpose + TAG_KIND_DELIMITER + "Lighting";
+    public static final String PUR_LIGHTING_ILLUMINATION = PUR_LIGHTING + TAG_KIND_DELIMITER + "Illumination";
+    public static final String PUR_LIGHTING_AMBIENCE = PUR_LIGHTING + TAG_KIND_DELIMITER + "Ambience";
+    public static final String PUR_LIGHTING_NIGHT = PUR_LIGHTING + TAG_KIND_DELIMITER + "Night";
+    public static final String PUR_HEATING = Kind.Purpose + TAG_KIND_DELIMITER + "Heating";
+    public static final String PUR_COOLING = Kind.Purpose + TAG_KIND_DELIMITER + "Cooling";
+
```

Comment:
I miss "Purpose:Presence" and "Purpose:Presence:Away" and "Purpose:Presence:AtHome".

Reply:
While reading the tags in the wiki I was unsure about why we added the "Presence" to "Purpose" **and** "Property", which is why I left them out of both. But thinking about it again now, something we **use** a device as a presence detector (for example a camera) and sometimes a device "measures" the presence, i.e. a motion detector... I will add them.